relay-server/src/actors/project_cache.rs

```diff
@@ -455,9 +461,60 @@ struct ProjectCacheBroker {
     /// Index of the buffered project keys.
     index: BTreeMap<ProjectKey, BTreeSet<QueueKey>>,
     buffer: Addr<Buffer>,
+    global_config: GlobalConfigStatus,
+}
+
+/// Status of global config needed for envelope processing.
```

Comment:
Maybe something like this will be better. 

```suggestion
/// Describes the current status of the [`GlobalConfig`], either it's ready to be used or contains the list of in-flight project keys, to be processed once the config arrives. 
```

Reply:
i agree, thanks!